subsys/bluetooth/controller/Kconfig

```diff
@@ -345,6 +322,7 @@ config BT_CTLR_DATA_LENGTH_MAX
 	int "Maximum data length supported"
 	depends on BT_CTLR_DATA_LENGTH
 	default 27
+	range 27 BT_BUF_ACL_RX_SIZE if BT_BUF_ACL_RX_SIZE < 251
```

Comment:
That's not as important, because on the TX path it is just an inefficient configuration as the host won't be able to queue the full data packet, but on the RX path it lead to invalid configuration since the length is not checked when copying.